test/src/api/commands/web-element/testGetText.js

```diff
@@ -110,4 +110,30 @@ describe('element().getText() command', function () {
     assert.strictEqual(await resultPromise.assert.not.contains('Signupx'), 'Signup');
     assert.strictEqual(await resultPromise.assert.not.matches(/Si[a-z]{2}upx/), 'Signup');
   });
+
+  it('test .element().find().text() assert', async function(){
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/1/text',
+      method: 'GET',
+      response: JSON.stringify({
+        value: 'Help'
+      })
+    }, true);
+
+    const resultPromise = this.client.api.element('#signupSection').find('#helpBtn').text();
+    assert.strictEqual(resultPromise instanceof Element, false);
+    assert.strictEqual(typeof resultPromise.find, 'undefined');
+
+    assert.strictEqual(resultPromise instanceof Promise, false);
+    assert.strictEqual(typeof resultPromise.then, 'function');
+
+    assert.strictEqual(await resultPromise.assert.equals('Help'), 'Help');
+    assert.strictEqual(await resultPromise.assert.contains('He'), 'Help');
+    assert.strictEqual(await resultPromise.assert.matches(/He[a-z]{1}/), 'Help');
+
+    assert.strictEqual(await resultPromise.assert.not.equals('Helpx'), 'Help');
+    assert.strictEqual(await resultPromise.assert.not.contains('Helpx'), 'Help');
+    assert.strictEqual(await resultPromise.assert.not.matches(/He[a-z]{1}x/), 'Help');
+  });
+
```

Comment:
This should be added as the second testcase of the test suite, with title "test .element().text() alias" and the contents of the test case would mostly be similar to the first test case.